docs/vultr.md

```diff
@@ -0,0 +1,31 @@
+# Variomedia
+
+## Configuration
+
+### Example
+
+```json
+{
+  "settings": [
+    {
+      "provider": "vultr",
+      "domain": "example.com",
+      "owner": "potato",
```

Comment:
The last program works with `domain` only, the owner (here `potato`) is automatically extracted from the domain string now.

```suggestion
      "domain": "potato.example.com",
```